cms/djangoapps/contentstore/views/preview.py

```diff
@@ -283,6 +283,7 @@ def _studio_wrap_xblock(xblock, view, frag, context, display_name_only=False):
         root_xblock = context.get('root_xblock')
         is_root = root_xblock and xblock.location == root_xblock.location
         is_reorderable = _is_xblock_reorderable(xblock, context)
+        is_not_library = not hasattr(xblock.location, 'library_key')
```

Comment:
I'm always a little wary of booleans that are negated; I think they require some brain-twists to understand the code. Perhaps reverse the logic here, i.e. call this `is_library`, and drop the `not` negation?

Reply:
Sure, that makes sense.